packages/server/lib/open_project.ts

```diff
@@ -174,7 +174,9 @@ export class OpenProject {
         return await browsers.connectToExisting(browser, options, automation)
       }
 
-      if (options.shouldLaunchNewTab) {
+      // if we should launch a new tab and we are not running in electron (which does not support connecting to a new spec)
+      // then we can connect to the new spec
+      if (options.shouldLaunchNewTab && browser.name !== 'electron') {
```

Comment:
we might need to check and some tests with https://github.com/cypress-io/cypress/pull/27116 or this (whatever goes in first) to make sure the behavior is expected, but I think the behavior should play nicely together.